### ISSUE TITLE
Remove hourly earnings countdown in frontend

### ISSUE THREAD

**Issue by poshdan** [CryptoBlades/cryptoblades#1137]
![image](https://user-images.githubusercontent.com/31659034/152629978-bab3d096-bbde-4e6d-a212-1bd66f9256ec.png)
Since we have moved on to use the multi skill for payouts, the hourly updates have been meaningless and only cause confusion.
It should be removed

**Comment by brian-man** [comment]
@poshdan
I agree with the potential confusion (and screen space consumption) concerns of this message, and preparing the PR but..

The game contract still makes interacts with `vars[VAR_HOURLY_TIMESTAMP]` in `updateHourlyPayouts()`.  Right?
